app/components/page-layout.hbs

```diff
@@ -7,17 +7,25 @@
       </LinkTo>
       {{#if this.showHeaderSearch}}
         <div class="search with-default-styling test-header-search">
-          <PowerSelectTypeahead @search={{perform this.searchForAddons}} @selected={{this.selectedAddon}} @placeholder="Search" @dropdownClass="test-header-search-dropdown" @noMatchesMessage="No matching addon names, try another search!" @onChange={{perform this.goToAddon}} as |addonName options|>
+          <PowerSelectTypeahead
+            @search={{perform this.searchForAddons}}
+            @selected={{this.selectedAddon}}
+            @placeholder="Search"
+            @dropdownClass="test-header-search-dropdown"
+            @noMatchesMessage="No matching addon names, try another search!"
+            @onChange={{perform this.goToAddon}}
+            as |addonName|
+          >
             {{#if addonName.isFullSearchLink}}
-              <LinkTo @route="index" @query={{hash query=options.searchText}} class="jump-to-full-search test-search-result-jump-to-full-search-link">
+              <span class="jump-to-full-search test-search-result test-search-result-jump-to-full-search-link">
```

Comment:
You've lost behavior here, however -- you can no longer right click -> open in new tab from the results in the typeahead.

Reply:
@kategengler I have reverted that commit and found no other solution to it. The problem is very weird:

When an `<a>` inside a PowerSelect's `<li>` is clicked, a hard redirect happens. The action attached to the `<a>` never fires.

The event target is `<a>`, while the event handler is attached to the `<li>`.

My guess is that it's the browser that's causing a hard redirect as it processes the click on `<a>`. But `event.preventDefault()` and `event.stopPropagation()` don't prevent this. :(

It only happens programmatically. I. e. when I replace `click()` with `pauseTest()`, click by hand and resume, the test passes.

I worked around the issue by replacing `click(find('a'))` with `click(find('a').parentElement)`. This is not a good solution as it's not what the user is actually doing. But it happens to resolve the issue, and the issue only happens in test.

If you have any other ideas to fix this, please tell me. One possibility is to upgrade Ember further, maybe it's a regression that has been fixed.

CC @mansona